schema/search.go

```diff
@@ -579,9 +556,9 @@ func (s *ImplicitSearchIndex) GetSearchDeltaFields(existingFields []*QueryableFi
 		tsFields = append(tsFields, tsApi.Field{
```

Comment:
We adding to the tsFields here but all the values would be `false` if we have no search fields. What would then happen for typesense? 
I was testing this on the background search branch and what I did was not add the field to the `tsFields` if there was no search field But this caused the crashes in collection creation

Reply:
All the `fields` false and `optional` true mean storing in search but not indexing. This is needed so we don't need to read from the database when we get search results. You should not change that because that will break the search as then search results won't have full data.